bitnami/external-dns/templates/servicemonitor.yaml

```
@@ -37,6 +37,14 @@ spec:
   {{- if .Values.metrics.serviceMonitor.jobLabel }}
   jobLabel: {{ .Values.metrics.serviceMonitor.jobLabel }}
   {{- end }}
+  {{- with .Values.metrics.serviceMonitor.podTargetLabels }}
+  podTargetLabels:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.metrics.serviceMonitor.targetLabels }}
+  targetLabels:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Could you please apply the following changes?
- Use `if` instead of `with`.
- Use the `common.tplvalues.render` instead of `toYaml` to allow template rendering.
- Set key and value in the same line
 
```suggestion
  {{- if .Values.metrics.serviceMonitor.podTargetLabels }}
  podTargetLabels: {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.serviceMonitor.podTargetLabels "context" $) | nindent 4 }}
  {{- end }}
  {{- if .Values.metrics.serviceMonitor.targetLabels }}
  targetLabels: {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.serviceMonitor.targetLabels "context" $) | nindent 4 }}
  {{- end }}
```